crypto/x509v3/v3_cpols.c

```diff
@@ -413,7 +413,8 @@ static void print_notice(BIO *out, USERNOTICE *notice, int indent)
             num = sk_ASN1_INTEGER_value(ref->noticenos, i);
             if (i)
                 BIO_puts(out, ", ");
-            tmp = i2s_ASN1_INTEGER(NULL, num);
+            if ((tmp = i2s_ASN1_INTEGER(NULL, num)) == NULL)
+                return;
```

Comment:
And @levitte is right that a straight return here isn't the right approach to take - we definitely don't do that elsewhere in this set of printing functions ...